api/server/model.cc

```diff
@@ -73,6 +73,10 @@ Cidr::Cidr() {
     mask_size = 0;
 }
 
+std::string Cidr::str() const {
```

Comment:
this can be merge into `std::hash<app::Cidr>::result_type std::hash<app::Cidr>::operator() (argument_type const& a)`